docs/development/license.rst

```diff
@@ -0,0 +1,28 @@
+.. _License:
+
+License
+=======
+
+This license is also available on the `project repository <https://github.com/HumanCompatibleAI/imitation/blob/master/LICENSE>`_.
```

Comment:
Can we use the `.include::` directive to avoid duplicating this? https://docutils.sourceforge.io/docs/ref/rst/directives.html#including-an-external-document-fragment